core/src/main/scala/kafka/server/ReplicaFetcherManager.scala

```diff
@@ -50,7 +50,7 @@ class ReplicaFetcherManager(brokerConfig: KafkaConfig,
         if (curMaxAll > curThreadMax) curMaxAll else curThreadMax
       })
     },
-    Map("clientId" -> "Replica")
+    Map("clientId" -> "Replica", "replica" -> "Follower")
```

Comment:
This looks better. Could we update the KIP to be up to date with the introduced metrics?